style.css

```diff
@@ -27,8 +27,10 @@ a {
 }
 
 .navbar-dark .navbar-nav .nav-link:focus,
-.navbar-dark .navbar-nav .nav-link:hover {
+.navbar-dark .navbar-nav .nav-link:hover,
+.nav-link {
     color: var(--font-color);
+    font-family: Get Schwifty;
```

Comment:
`nav-link` added to the style sheet. Links in the `navbar` now render with the 'Get Schwifty' font. This looks pretty good, but creates a bit of discrepancy between the navigation bar and the page content.